.travis.yml

```diff
@@ -33,7 +33,7 @@ matrix:
       sudo: required
 
 before_install:
-  - npm install -g npm@latest
+  - npm install -g npm@5.3.0
```

Comment:
@jeddy3 Could you explain why you made this change?

FYI: `npm install -g npm@latest` is valid and the recommended command to [update npm](https://docs.npmjs.com/all#updating-npm)

p.s. The current npm release is `5.4.0`

Reply:
Yep, sure. I was trying to get this branch to pass by pinning the npm version to 5.3.0 as there seems to be [an issue](https://github.com/facebook/flow/issues/4733) with flow and 5.4.0. Unfortunately, I ran out of time to finish this up. Feel free to revert or pick this up.

Reply:
Thanks, makes sense, happy to keep npm pinned at 5.3.0 until upstream issue is resolved

Reply:
As this is causing issues with contributors pull requests I've created #2849 and merged this change into `master` via commit https://github.com/stylelint/stylelint/commit/5d75094285bea5a6cc4c218a61c6a7d48244e2e7 just now.

Reply:
@ntwb Thanks!